src/table.jl

```diff
@@ -570,10 +570,9 @@ function buildbitmap(batch, rb, nodeidx, bufferidx)
 end
 
 function uncompress(ptr::Ptr{UInt8}, buffer, compression)
-    if buffer.length == 0
-        return 0, UInt8[]
-    end
+    buffer.length == 0 && return 0, UInt8[]
```

Comment:
maybe we should use iszero instead?

```suggestion
    iszero(buffer.length) && return 0, UInt8[]
```